wordle-web/components/GameBoardGuess.vue

```diff
@@ -16,15 +21,17 @@ const props = defineProps<{
 
 const isShaking = ref(false);
 
-watch(() => props.guess.isFilled,
-() => {
-  if(!props.guess.isFilled){
-    isShaking.value = true;
-    setTimeout(() => {
-      isShaking.value = false;
-    }, 300);
+watch(
+  () => props.guess.isFilled,
+  () => {
+    if (!props.guess.isFilled) {
+      isShaking.value = true;
```

Comment:
```suggestion
watch(
  () => props.guess.isFilled,
  (filled) => {
    if (filled) {
      isShaking.value = true;
```
When implementing watchers, the call back can implement a couple parameters `(newValue, oldValue) => { }`... in this case we don't care about the old value, but it's good practice to use these for a couple reason:
- Can improve clarity (this is up for debate), and reduce verbosity.
- Although it seems like this is doing the same thing, it ensures the logic is based on the updated value instead of accessing props.guess.isFilled again, as asynchronicity leaves opportunity for the value to change before this is evaluated and can lead to unexpected behavior.